src/context/AuthContext.jsx

```diff
@@ -37,16 +40,23 @@ function AuthProvider({ children }) {
         localStorage.setItem("token", encodedToken);
         authDispatch({ type: SIGN_IN, payload: { token: encodedToken } });
         navigate("/");
+        toast.success("Signed in");
```

Comment:
Suggestion: You can display the name of the user in the toast. For eg, **Welcome, Saran!**